QuasarHelios2015/src/org/team1540/quasarhelios/AutoEjector.java

```diff
@@ -35,9 +35,10 @@ public static BooleanStatus create() {
     protected void autonomousMain() throws AutonomousModeOverException, InterruptedException {
         try {
             setClampHeight(1.0f);
-            Elevator.setBottom.event();
-
-            waitUntil(Elevator.atBottom);
+            if (!Elevator.atBottom.get()) {
+                Elevator.setBottom.event();
+                waitUntil(Elevator.atBottom);
+            }
```

Comment:
If ``Elevator.atBottom`` isn't true when it is already at the bottom, then how does this fix the problem? You are still checking to see if ``Elevator.atBottom`` is true.